### DISCUSSION TITLE
Text Caption with Loading Overlay

### DISCUSSION THREAD

**Discussion by kimfucious** [mantinedev/mantine#4021] [Feature requests]
I am admittedly super lazy, which is why I love how much work Mantine saves me.

The loading overlay is super helpful, but there are times when I'd like to be able to display some text in addition to a spinner.

That would be a nice feature that maybe others might like too.

**Comment by Vaibhav-Kulkarni-4** [comment]
You can make use of the [custom loader prop](https://v6.mantine.dev/core/loading-overlay/#custom-loader) to create an element that will have the loader logo along with the text. 

Something like this - 

![image](https://github.com/mantinedev/mantine/assets/37895289/5d8adf43-2950-4105-a874-39f12c873209)

**Comment by kimfucious** [comment]
Thanks, @Vaibhav-Kulkarni-4.

I like the loaders as they are and don't know the SVG settings to use in a custom loader.

I was thinking that a`label` prop would be quite useful.   

Something like this:

```ts
<LoadingOverlay loaderProps={{ color: "green", label: "Hey Buddy" }} visible />;
```

**Reply by Vaibhav-Kulkarni-4** [reply]
Hey @kimfucious,

Yes, I understand. This of course can be taken as a feature enhancement by Mantine. But for now, the workaround for me was via the custom loader prop only.

**Reply by kimfucious** [reply]
I appreciate your input; thanks.
